pkg/scheduler/scheduler.go

```diff
@@ -459,6 +459,7 @@ func (sched *Scheduler) scheduleOne() {
 			metrics.PreemptionAttempts.Inc()
 			metrics.SchedulingAlgorithmPremptionEvaluationDuration.Observe(metrics.SinceInMicroseconds(preemptionStartTime))
 		}
+		metrics.PodScheduleFailures.Inc()
```

Comment:
This is not accurate. We should increment `PodScheduleFailures` if the returned error is a "FitError". Otherwise, we should increment `PodScheduleErrors`.